ui-ngx/src/app/modules/home/components/profile/device/lwm2m/lwm2m-device-profile-transport-configuration.component.ts

```diff
@@ -214,6 +219,8 @@ export class Lwm2mDeviceProfileTransportConfigurationComponent implements Contro
         this.updateObserveAttrTelemetryObjects(telemetryArray, clientObserveAttrTelemetry, TELEMETRY);
       }
       if (isDefinedAndNotNull(keyNameJson)) {
+        this.configurationValue.observeAttr.keyName = deepClone(this.validateKeyNameObjects(keyNameJson, attributeArray, telemetryArray));
```

Comment:
Why is `deepClone` called when a result function `validateKeyNameObjects`  is a new object?